### ISSUE TITLE
Replacing string with table corrupts DOCX

### ISSUE THREAD

**Issue by LucasBornhauser** [gyselroth/docxbox#42]
**Describe the bug**
Replacing a string with a table makes DOCX invalid

**To Reproduce**
1. Command: 
-  docxbox rpt table_unordered_list_images.docx Officia "{\"table\":{\"header\":[\"A\",\"B\",\"C\"],\"content\":[[\"a1\",\"a2\",\"a3\"],[\"b1\",\"b2\",\"b3\"],[\"c1\",\"c2\",\"c3\"]]}}"

2. Resources: table_unordered_list_images.docx

**Expected behavior**
- Given string is replaced by table (works)
- DOCX is valid →DOCX is invalid 

**Screenshots**
![Auswahl_059](https://user-images.githubusercontent.com/32673972/83146130-21577480-a0f6-11ea-9592-3b83e786c0c4.png)

![word_errormessage](https://user-images.githubusercontent.com/32673972/83147072-7a73d800-a0f7-11ea-9330-c15125306e8d.png)

**Environment:**
 - docxBox Version: 0.0.5
 - DOCX Processor: Word 2019

**Additional information**
Screenshot only shows relevant lines, the actual diff has more lines

**Comment by kstenschke** [comment]
@LucasBornhauser  Can the resulting docx be opened e.g. in FreeOffice TextMaker w/o corruption message?

**Comment by LucasBornhauser** [comment]
Yes, all other (tested) DOCX processor manage to open the DOCX: 
- Abi Word: Document can be opened, table borders are not displayed, but the content is
- LibreOffice: Document can be opened, no table or table content is displayed
- FreeOffice 2018 TextMaker: Document can be opened, table and content are displayed

No error message is given upon opening the file with either one of these applications
